### PR TITLE
fix: Solving Ticket Sales ambiguity

### LINKED ISSUE #2796: Ambiguity in 'Events/Overview' Sales Section
**Describe the bug**

Sales total shown in `Event/Overview` is different in Overview and Each Ticket section.

**To Reproduce**

Steps to reproduce the behaviour:

1. Go to `Events/Overview` 
2. See error
3. Refer to Screenshot

**Expected behaviour**

Ambiguity should not be there

**Screenshots**

image

**Desktop (please complete the following information):**
 - OS: MacOS Mojave
 - Browser: Safari
 - Version: 23.0.3

**Additional context**

Looking into the cause

### PR DESCRIPTION
**Pull request by kushthedude**
Fixed #2796

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 @uds5501 @shreyanshdwivedi @mrsaicharan1 Please Review

File: app/templates/events/view/tickets/index.hbs [hunk 1]
```diff
@@ -150,15 +150,15 @@
...
           {{#unless device.isMobile}}
             <td class="right aligned">{{ticket.orderStatistics.tickets.placed}}</td>
           {{/unless}}
-          <td class="right aligned">{{currency-symbol model.eventDetail.paymentCurrency}} {{format-number ticket.orderStatistics.sales.placed}}</td>
+          <td class="right aligned">{{currency-symbol model.eventDetail.paymentCurrency}} {{format-number (mult ticket.orderStatistics.tickets.placed ticket.price)}}</td>
           {{#unless device.isMobile}}
             <td class="right aligned">{{ticket.orderStatistics.tickets.pending}}</td>
           {{/unless}}
-          <td class="right aligned">{{currency-symbol model.eventDetail.paymentCurrency}} {{format-number ticket.orderStatistics.sales.pending}}</td>
+          <td class="right aligned">{{currency-symbol model.eventDetail.paymentCurrency}} {{format-number (mult ticket.orderStatistics.tickets.pending ticket.price)}}</td>
```
Comment (shreyanshdwivedi) [inline app/templates/events/view/tickets/index.hbs]: There can be decimal values too, right? The helper format-number will make them integer.

**Comment by kushthedude** [comment]
@CosmicCoder96 @shreyanshdwivedi @uds5501 @shreyanshdwivedi Please REview

**Comment by kushthedude** [comment]
@CosmicCoder96 Please REview

**Comment by kushthedude** [comment]
@mrsaicharan1 @shreyanshdwivedi Please REview

**Comment by kushthedude** [comment]
@mrsaicharan1 @shreyanshdwivedi Please REview

**Pull request merged**